src/AdoNetCore.AseClient/AseError.cs

```diff
@@ -1,9 +1,11 @@
-﻿namespace AdoNetCore.AseClient
+﻿using System;
+
+namespace AdoNetCore.AseClient
 {
     /// <summary>
     /// Collects information relevant to a warning or error returned by the data source.
     /// </summary>
-    public sealed class AseError 
+    public sealed class AseError : IDisposable
```

Comment:
This can’t  be right? Is the Sybase AseError also IDisposable?

Reply:
Yep, it is

Reply:
Dang. Alrighty then.